src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  const resArr = [];
+  let copyState = { ...state };
+
+  for (const instr of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
better to name it in this way, it's clearer